requirements.txt

```diff
@@ -0,0 +1 @@
+awscli==1.7.25
```

Comment:
Well if we want users to add this to your rails app then that could be an issue.

On Mon, May 4, 2015 at 3:30 PM, Ken Liu <notifications@github.com> wrote:

>> @@ -0,0 +1 @@
>> +awscli==1.7.25
> I believe Heroku detects the stack based on whether it sees `requirements.txt` or `Gemfile`. So if it sees `requirements.txt` then it will treat the app as a python app.
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/kbaum/heroku-database-backups/pull/5/files#r29614877